api/public/cerbos/policy/v1/policy.proto

```diff
@@ -230,7 +230,12 @@ message Match {
 }
 
 message Output {
+  message On {
+    string condition_not_met = 1;
+  }
+
   string expr = 1;
+  On on = 2;
```

Comment:
Let's structure this like the following. To keep backward compatibility, we can copy the value of `expr` to `on.condition_met`. Everywhere else in the code we should just use `on.condition_met`. We should also update the documentation to show the new way of doing it.

```suggestion
  message On {
    string condition_met = 1;
    string condition_not_met = 2;
  }

  string expr = 1 [deprecated = true];
  On on = 2;
```